ZipBuilder/Sources/ZipBuilder/CocoaPodUtils.swift

```diff
@@ -21,16 +21,28 @@ import Foundation
 public enum CocoaPodUtils {
   // MARK: - Public API
 
-  /// Information associated with an installed pod.
-  public struct PodInfo {
+  public struct VersionedPod {
     /// Public name of the pod.
-    var name: String
+    let name: String
 
     /// The version of the pod.
-    var version: String
+    let version: String
```

Comment:
Sorry should have caught this before - do we want to make version optional now since it's not always required when requesting a pod? I'm open to either but wanted to check with you.

Reply:
Done. nil is more Swifty than an empty string.